components/learn/modules/ROOT/pages/unit-testing.adoc

```diff
@@ -0,0 +1,126 @@
+= Testing Smart Contracts
+
+In a blockchain environment, a single mistake could cost you all of your funds (or even worse, your users'!). This guide will teach you how to write automated tests that verify your application behaves exactly as you intended.
+
+== About Testing
```

Comment:
I'd drop this section, assuming that the reader already knows what a test is. We can make some clarifications about unit vs e2e tests, and ganache vs regular nodes (taking a page out of [React's docs](https://reactjs.org/docs/testing.html)).